src/Platforms/SqlitePlatform.php

```diff
@@ -328,14 +330,34 @@ protected function _getCommonIntegerTypeDeclarationSQL(array $column)
         return ! empty($column['unsigned']) ? ' UNSIGNED' : '';
     }
 
+    /**
+     * Disables schema emulation.
+     *
+     * Schema emulation is enabled by default to maintain backwards compatibility.
+     * Disable it to opt-in to the behavior of DBAL 4.
+     *
+     * @deprecated Will be removed in DBAL 4.0.
+     */
+    public function disableSchemaEmulation(bool $reenable = false): void
```

Comment:
Because the flag should be settable from somewhere else by boolean and my preference is still the method name proposed by @derrabus.

But I will do what you want, my primary goal is to have this method present.

Reply:
Done, I got it, you want it for something like:

```
try {
    $platform->disableSchemaEmulation();
    // do some work here /wo emulation
} finally {
    $platform->enabledSchemaEmulation();
}
```